confgenerator/testdata/invalid/linux/metrics-processor_exclude_metrics_invalid_parameter_metric_prefixes/golden_error

```diff
@@ -1,9 +1,9 @@
-[6:7] "metrics_pattern[0]" must end with "/*","metrics_pattern[1]" must start with "agent.googleapis.com/"
+[6:7] "metrics_pattern[0]" must end with "/*","metrics_pattern[1]" must start with "agent.googleapis.com/","metrics_pattern[2]" must be a metric type glob pattern
```

Comment:
Shall this be [6:8]? As you added a new metrics pattern in here on line 8 which is wrong in syntax

Reply:
That is an excellent question, but this is, in fact, the exact error message produced by the validation library (generated via `-update_golden`). There may be a bug in our interface to the `yaml` library's validator functionality. Perhaps @quentinmit has some insight here…

Reply:
Ohh. Never mind. The "7" in "[6:7]" is the _column_, not the end line of the range ([code](https://github.com/goccy/go-yaml/blob/master/internal/errors/error.go#L186)). Looks like we don't actually track the range of lines for multiple errors.